### PR TITLE
docs: add dual region backup

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/backup-and-restore.md [hunk 1]
```diff
@@ -6,81 +6,110 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
-        It is recommended to schedule automatic backup at a low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+6. Click **Confirm** again to save changes.
 
-    - In **Backup Retention**, configure the minimum backup data retention period.
+### Configure backup schedule
 
-    - Turn on or off the PITR (**Point-in-time Recovery**) feature.
+  > **Warning**
+  >
+  > - When weekly backup is enabled, point-in-time restore will be enabled by default and cannot be modified.
```
Comment (grovecai) [inline tidb-cloud/backup-and-restore.md:60]: 这个跟上面的声明重复， 都要保留么？

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -6,81 +6,110 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
 
-    If you turn on PITR, you can select the **Perform a backup immediately and use it as recovery starting point in PITR.** checkbox. Otherwise, PITR will not be available until the next backup is completed.
 
-5. Click **Confirm**.
+5. In **Backup Retention**, configure the minimum backup data retention period. The default is 7 days.
 
-### Backup storage region support
+    > **Note**
+    >
+    > - The automatic backup files will be retained for 31 days from the date of cluster deletion. It is advisable to delete the backup files accordingly.
```
Comment (grovecai) [inline tidb-cloud/backup-and-restore.md:85]: 这个是不对， 集群删除之后， 自动备份的回收周期还是按照之前的retention 设置。

File: tidb-cloud/backup-and-restore.md [hunk 3]
```diff
@@ -147,27 +171,27 @@ To restore your TiDB Dedicated cluster data from a backup to a new cluster, take
 
     To restore a selected backup to the new cluster, take the following steps:
 
-    1. Click **Select Backup Name**.
-    2. Select a backup you want to restore to.
+    3. Click **Select Backup Name**.
```
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md:174]:
```suggestion
    1. Click **Select Backup Name**.
```

File: tidb-cloud/backup-and-restore.md [hunk 4]
```diff
@@ -147,27 +171,27 @@ To restore your TiDB Dedicated cluster data from a backup to a new cluster, take
...
+    4. Select a backup you want to restore to.
```
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md:175]:
```suggestion
    - Select a backup you want to restore to.
```

File: tidb-cloud/backup-and-restore.md (hunk 2 above)
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md:85]:
```suggestion
    > - After you delete the cluster, the automatic backup files will be kept for a specific duration, as set in Backup Settings. Remember to delete these backup files when necessary.
```

**Review by grovecai — APPROVED** [review]
Rest LGTM

**Comment by grovecai** [comment]
pls fix pricing page also as specified in https://pingcap.feishu.cn/wiki/Vh6rwk24biSyvRkCyA0ci818ngh

File: tidb-cloud/backup-and-restore.md [hunk 5]
```diff
@@ -6,81 +6,110 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
 >
 > To learn how to back up and restore TiDB Serverless cluster data, see [Back Up and Restore TiDB Serverless Data](/tidb-cloud/backup-and-restore-serverless.md).
 
 ## Limitations
 
 - TiDB Cloud does not support restoring tables in the `mysql` schema, including user permissions and system variables.
-- If you turn on and off PITR (Point-in-time Recovery) multiple times, you can only choose a time point within the recoverable range after the most recent PITR is enabled. The earlier recoverable range is not accessible.
 
-## Backup
+- If you turn on and off Point-in-time Restore (Point-in-time Restore) multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:19]:
```suggestion
- If you turn on and off Point-in-time Restore multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
```

File: tidb-cloud/backup-and-restore.md [hunk 6]
```diff
@@ -6,81 +6,110 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
 
-TiDB Dedicated supports automatic backup and manual backup.
+## Turn on auto backup
 
-Automatic backups are scheduled for your TiDB Dedicated clusters according to the backup setting, which can reduce your loss in extreme disaster situations. You can also pick a backup snapshot and restore it into a new TiDB Dedicated cluster at any time.
+TiDB Dedicated supports both [snapshot backups](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backups](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backup enables users to restore to the the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:23]:
```suggestion
TiDB Dedicated supports both [snapshot backups](https://docs.pingcap.com/tidb/stable/br-snapshot-guide) and [log backups](https://docs.pingcap.com/tidb/stable/br-pitr-guide). Snapshot backups enable you to restore data to the backup point. By default, snapshot backups are taken automatically and stored according to your backup retention policy.
```

File: tidb-cloud/backup-and-restore.md [hunk 7]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
+> **Tip**
 >
 > To learn how to back up and restore TiDB Serverless cluster data, see [Back Up and Restore TiDB Serverless Data](/tidb-cloud/backup-and-restore-serverless.md).
 
 ## Limitations
 
 - TiDB Cloud does not support restoring tables in the `mysql` schema, including user permissions and system variables.
-- If you turn on and off PITR (Point-in-time Recovery) multiple times, you can only choose a time point within the recoverable range after the most recent PITR is enabled. The earlier recoverable range is not accessible.
+- If you turn on and off Point-in-time Restore multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
+- Do modify the switch of **Point-in-time Restore** and **Dual Region Backup** at the same time.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
- Do modify the switches of **Point-in-time Restore** and **Dual Region Backup** at the same time.
```

Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:19]:
```suggestion
- DO NOT modify the switch of **Point-in-time Restore** and **Dual Region Backup** at the same time.
```

File: tidb-cloud/backup-and-restore.md [hunk 8]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
 
-    - Turn on or off the PITR (**Point-in-time Recovery**) feature.
+    - In **Backup Retention**, configure the minimum backup data retention period. The default is seven days.
 
-        > **Note:**
+        > **Note**
         >
-        > - The PITR feature is enabled by default and cannot be disabled when you enable weekly backup.
-        > - If you change the backup cycle from weekly to daily, the PITR feature remains enabled. You can manually disable it if needed.
+        > - After you delete the cluster, the automatic backup files will be retained for a specified period, as set in **Backup Settings**. You need to delete the backup files accordingly.
```
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md:95]:
```suggestion
        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
```

File: tidb-cloud/backup-and-restore.md [hunk 9]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
+
+5. Click **Confirm** to preview the configuration changes.
+
+6. Click **Confirm** again to save changes.
+
+#### Configure backup schedule
+
+TiDB Dedicated supports daily and weekly backup schedules. By default, the backup schedule is set to daily. You can choose a specific time of the day or week to start snapshot backup.
+
+To configure the backup schedule, toggle the **Auto Backup** switch to **On** and make the following settings:
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:63]:
```suggestion

To configure the backup schedule, perform the following steps:
```

File: tidb-cloud/backup-and-restore.md [hunk 10]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
+        If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
 
-    - In **Backup Retention**, configure the minimum backup data retention period.
+        > **Note**
+        >
+        > - To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
+        > - Backup jobs are automatically delayed when data import jobs are in progress. **DO NOT** run manual backups during data import, or during cluster scaling.
 
-    - Turn on or off the PITR (**Point-in-time Recovery**) feature.
+    - In **Backup Retention**, configure the minimum backup data retention period. The default is seven days.
```
Comment (benmaoer) [inline tidb-cloud/backup-and-restore.md:91]:
```suggestion
    - In **Backup Retention**, configure the minimum backup data retention period. The default is 7 days. To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
```

File: tidb-cloud/backup-and-restore.md [hunk 11]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
     - In **Backup Time**, schedule a start time for the daily or weekly cluster backup.
 
-        It is recommended to schedule automatic backup at a low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+        If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
 
-    - In **Backup Retention**, configure the minimum backup data retention period.
+        > **Note**
+        >
+        > - To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
+        > - Backup jobs are automatically delayed when data import jobs are in progress. **DO NOT** run manual backups during data import, or during cluster scaling.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:86]:
```suggestion
        > - Backup jobs are automatically delayed when data import jobs are in progress. **DO NOT** run manual backups during data import or cluster scaling.
```

File: tidb-cloud/backup-and-restore.md (hunk 10 above)
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
    - In **Backup Retention**, configure the minimum backup data retention period. The default period is seven days.
```

**Review by benmaoer — COMMENTED** [review]
need changes

File: tidb-cloud/backup-and-restore.md [hunk 12]
```diff
@@ -6,81 +6,156 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
-| Google Cloud            | Tokyo (asia-northeast1)     | Osaka (asia-northeast2)  |
+### Turn off auto backup
 
-> **Note:**
+> **Note**
 >
-> If you select multiple backup storage regions, you will be charged for multiple backup storage and inter-region backup data replication out from the cluster region to each destination region. The cost is on a per-region basis and varies with the backup regions selected. For more information, see [Data Backup Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#backup-storage-cost).
+> Turning off auto backup will also turn off point-in-time restore by default.
+
+To turn off auto backup:
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:125]:
```suggestion
To turn off auto backup, perform the following steps:
```

File: tidb-cloud/backup-and-restore.md [hunk 13]
```diff
@@ -131,44 +194,54 @@ To restore your TiDB Dedicated cluster data from a backup to a new cluster, take
...
+    >
+    > - The default value of the **Restore From Region** is the same as the backup cluster.
+
+4. In **Restore Mode**, choose to restore data of any point in time or a selected backup to a new cluster.
 
     <SimpleTab>
     <div label="Select Time Point">
 
-    To restore data of any point in time within the backup retention to a new cluster, make sure that **PITR** in **Backup Settings** is on and then take the following steps:
+    To restore data of any point in time within the backup retention to a new cluster, make sure that **Point-in-time restore** in **Backup Settings** is on and then take the following steps:
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:208]:
```suggestion
    To restore data of any point in time within the backup retention to a new cluster, make sure that **Point-in-time Restore** in **Backup Settings** is on and then take the following steps:
```

File: tidb-cloud/backup-and-restore.md [hunk 14]
```diff
@@ -6,81 +6,153 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
+> **Tip**
 >
 > To learn how to back up and restore TiDB Serverless cluster data, see [Back Up and Restore TiDB Serverless Data](/tidb-cloud/backup-and-restore-serverless.md).
 
 ## Limitations
 
 - TiDB Cloud does not support restoring tables in the `mysql` schema, including user permissions and system variables.
-- If you turn on and off PITR (Point-in-time Recovery) multiple times, you can only choose a time point within the recoverable range after the most recent PITR is enabled. The earlier recoverable range is not accessible.
+- If you turn on and off Point-in-time Restore multiple times, you can only choose a time point within the recoverable range after the most recent Point-in-time Restore is enabled. The earlier recoverable range is not accessible.
+- DO NOT modify the switch of **Point-in-time Restore** and **Dual Region Backup** at the same time.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:19]:
```suggestion
- DO NOT modify the switche of **Point-in-time Restore** and **Dual Region Backup** at the same time.
```

File: tidb-cloud/backup-and-restore.md [hunk 15]
```diff
@@ -6,81 +6,153 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
...
-        It is recommended to schedule automatic backup at a low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+        If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
 
-    - In **Backup Retention**, configure the minimum backup data retention period.
+        > **Note**
+        >
+        > - Backup jobs are automatically delayed when data import jobs are in progress. **DO NOT** run manual backups during data import, or during cluster scaling.
 
-    - Turn on or off the PITR (**Point-in-time Recovery**) feature.
+    - In **Backup Retention**, configure the minimum backup data retention period. The default is 7 days. To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:88]:
```suggestion
    - In **Backup Retention**, configure the minimum backup data retention period. The default period is 7 days. To minimize the impact on business, it is recommended to schedule automatic backup during periods of low workloads.
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**